adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -59,7 +59,7 @@ import org.bdgenomics.utils.io.LocalFileByteAccess
 import org.bdgenomics.utils.misc.HadoopUtil
 import org.bdgenomics.utils.misc.Logging
 import org.seqdoop.hadoop_bam._
-import org.seqdoop.hadoop_bam.util.{ BGZFCodec, SAMHeaderReader, VCFHeaderReader, WrapSeekable }
+import org.seqdoop.hadoop_bam.util.{ BGZFCodec, BGZFEnhancedGzipCodec, SAMHeaderReader, VCFHeaderReader, WrapSeekable }
```

Comment:
Small nit for readability: can you change this to either `org.seqdoop.hadoop_bam.util._` or split it up across multiple lines? I think we typically go to importing the whole package at 4 imports.